ejson/ejson.d.ts

```diff
@@ -0,0 +1,24 @@
+// Type definitions for ejson v2.1.2
+// Project: https://www.npmjs.com/package/ejson
+// Definitions by: Shantanu Bhadoria <https://github.com/shantanubhadoria>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare interface IStringifyOptions {
```

Comment:
Got it! Let me make the changes and get back to you. tslint seems to force the "I" prefix in its default configuration. Is there a tslint configuration that you use if you don't mind? 
Thanks!